src/Rbac/Rbac.php

```diff
@@ -180,6 +181,11 @@ protected function _matchPermission(array $permission, $user, $role, ServerReque
                 $return = (bool)call_user_func($value, $user, $role, $request);
             } elseif ($value instanceof Rule) {
                 $return = (bool)$value->allowed($user, $role, $request);
+            } elseif (is_array($value) && array_key_exists('className', $value)) {
+                $rule = RuleRegistry::get($value['className'], $value['options'] ?? []);
```

Comment:
Shouldn't we get an error if not used a valid class here?

Reply:
We can do it.

Reply:
I agree. This is the last comment to fix before merge. Thank you @skie